### PR TITLE
[11.0][ADD] web_widget_x2many_2d_matrix: add option to set label for total column

### PR DESCRIPTION
**Pull request by vincent-hatakeyama**

### CODE REVIEW

**Comment by vincent-hatakeyama** [comment]
Concerning the CLA, it has been signed by my company.

**Review by daramousk — APPROVED** [review]
Looks great, perhaps an extra addition would be `label_column_totals`. Travis issues need to be addressed of course

File: web_widget_x2many_2d_matrix/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "2D matrix for x2many fields",
-    "version": "11.0.1.1.2",
+    "version": "11.0.1.1.3",
```
Comment (simahawk) [inline web_widget_x2many_2d_matrix/__manifest__.py]: this is a feature, not a patch. Please revert this change: we can handle proper number by merging w/ OCA bot ;)

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
Travis error is related to linting https://travis-ci.org/OCA/web/jobs/535826166#L1534
